### DISCUSSION TITLE
Traced installs possible for purpose of uninstalls - Bulk Crap Uninstaller

### DISCUSSION THREAD

**Discussion by perfection161** [Klocman/Bulk-Crap-Uninstaller#362] [Q&A] [answered]
Does / Can this program (BC Uninstaller) monitor software installs to do the uninstalls ?

Is such a feature feasible to request for and will there be a major difference in the "cleanliness' of uninstalls compared to how unistalls are traced/tracked currently by BCU?

**Comment by joesmith222** [comment]
i would like this too if it was possible

**Comment by Klocman** [comment, marked as answer]
Check https://github.com/Klocman/Bulk-Crap-Uninstaller/issues/258
